bootstrap.sh

```diff
@@ -224,14 +224,30 @@ fi
 echo "Installing dependencies for building web UI"
 angular_cli_dir=$VTROOT/dist/angular-cli
 web_dir2=$VTTOP/web/vtctld2
-rm -rf $angular_cli_dir
-cd $VTROOT/dist && git clone https://github.com/angular/angular-cli.git --quiet
-cd $angular_cli_dir && git checkout 3dcd49bc625db36dd9f539cf9ce2492107e0258c --quiet
+
+if [[ -d $angular_cli_dir ]]; then
+  echo "skipping angular cli download. remove $angular_cli_dir to force download."
+else
+  cd $VTROOT/dist && git clone https://github.com/angular/angular-cli.git --quiet
+  cd $angular_cli_dir && git checkout 3dcd49bc625db36dd9f539cf9ce2492107e0258c --quiet
+fi
 cd $angular_cli_dir && $node_dist/bin/npm link --silent
-$node_dist/bin/npm install -g bower --silent
-cd $web_dir2 && $node_dist/bin/npm install --silent
+if [[ -f $node_dist/bin/bower ]]; then
+  echo "skipping npm install of bower. Remove $node_dist/bin/bower to force install."
+else
+  $node_dist/bin/npm install -g --prefix $node_dist bower --silent
+fi
+if [[ -d $web_dir2/node_modules ]]; then
```

Comment:
Hmm so what's a reasonable alternative?  It certainly takes a long time to do an npm install (about two minutes) and you don't want to make everyone who bootstraps redo that step.